### ISSUE TITLE
Symlinks defeat the language server

### ISSUE THREAD

**Issue by mpickering** [microsoft/vscode-lsif-extension#5]
If I symlink a file to the workspace then the language server refuses to read it. It only works if I copy the `lsif.json` file into the right place.

**Comment by dbaeumer** [comment]
@mpickering you symlinked the `lsif.json` file. Right?

**Comment by mpickering** [comment]
Yes I made a symlink from `test.json` to `lsif.json` and the language server failed to load it.

**Comment by dbaeumer** [comment]
This changes with the new version of teh extension which lets you point to a specific json file (regardless of its name).
